src/gc/windows/gcenv.windows.cpp

```diff
@@ -30,6 +30,191 @@ typedef BOOL (WINAPI *PQUERY_INFORMATION_JOB_OBJECT)(HANDLE jobHandle, JOBOBJECT
 
 namespace {
 
+static bool g_fEnableGCNumaAware;
+
+struct CPU_Group_Info 
+{
+    WORD    nr_active;  // at most 64
+    WORD    reserved[1];
+    WORD    begin;
+    WORD    end;
+    DWORD_PTR   active_mask;
+    DWORD   groupWeight;
+    DWORD   activeThreadWeight;
+};
+
+static bool g_fEnableGCCPUGroups;
+static bool g_fHadSingleProcessorAtStartup;
+static DWORD  g_nGroups;
+static DWORD g_nProcessors;
+static CPU_Group_Info *g_CPUGroupInfoArray;
+
+void InitNumaNodeInfo()
+{
+    ULONG highest = 0;
+    
+    g_fEnableGCNumaAware = false;
+
+    if (!g_fIsNumaAwareEnabledByConfig)
```

Comment:
Why not just call `GCConfig::GetGCNumaAware` here directly?

Reply:
Thanks for catching this. This was a (somewhat hacky) workaround trying to use the same code for unix and windows. It's not needed at this point.